RecoParticleFlow/PFRecHitProducer/interface/PFRecHitParamsRecord.h

```diff
@@ -0,0 +1,14 @@
+#ifndef RecoParticleFlow_PFRecHitProducer_ParamsRecord_h
+#define RecoParticleFlow_PFRecHitProducer_ParamsRecord_h
+
+#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "CondFormats/DataRecord/interface/EcalPFRecHitThresholdsRcd.h"
+
+class PFRecHitHCALParamsRecord : public edm::eventsetup::EventSetupRecordImplementation<PFRecHitHCALParamsRecord> {};
```

Comment:
If this record will always be used only to copy parameters from the python configuration to the event setup (to make them available on GPU), you could reuse the `JobConfigurationGPURecord` instead.

Currently it's defined in `HeterogeneousCore/CUDACore/interface/JobConfigurationGPURecord.h`, but we could move it to a place that is not dependent on CUDA.

Reply:
There is an effort to move these parameters from the configuration to the database. But it is not yet implemented in the serial version of the producer, so I also did not do it here.
Furthermore, similar to ECAL, other types of cut might be added in the future.

Reply:
ok